### ISSUE TITLE
Crius AIOP v1.1 with megapirateng 3.0.1 R4, red led blinks twice rapidly

### ISSUE THREAD

**Issue by auguste95** [MegaPirateNG/ardupilot-mpng#141]
Hi,
First I would like to apologize for my bad english (I'm french).

So the problem is :

After compiling the code (Megapirateng, 3.0.1 R4, the last version in fact) on my Crius AIOP v1.1, the red led blinks 2 times rapidly (ta ta --- ta ta --- ta ta --- ta ta) and I can't arm the motors.

In fact, I think that the board doesn't detect the radio, because on Mission planner I can't calibrate it. However, the receiver detects the radio because a red led is shining (which means it detects it).
The radio is a turnigy 9x, flashed with ER9X.

And there is no problem with older version like 2.8 R3.
I also tried to use flashtool, but it doesn't work (same result as before).

I hope someone will have the solution, because nobody have one on french forums.

Thank you very much, 

auguste95.

**Comment by sosko71** [comment]
Did you use Flashtool or Arduino. I think that you did not select proper type of RC Input. You have to use PWM input with Multiwii configuration.

**Comment by auguste95** [comment]
I tried the two ways. I think so, that's why I'm going to retry with flashtool and arduino.
I give you the resulsts as soon as possible. 

Thank you for helping me.

**Comment by auguste95** [comment]
Hi

I retried arduino and flashtool, but it didn't worked. 

But I found on the APM copter website the solution : 
In fact th board didn't armed because its pre-arming checks weren't all verified.
Everything is well explained on this page : http://copter.ardupilot.com/wiki/prearm_safety_check/
I could calibrate magnetometer, receiver, radio, gimbal etc...

Now everything works good, 
thank you very much for your help,
have a good day (it's raining here),
auguste95.
